frontend/public/actions/ui.ts

```diff
@@ -125,6 +130,11 @@ export const getNodeMetric = (node: NodeKind, metric: string): number => {
   return metrics?.[metric]?.[node.metadata.name] ?? 0;
 };
 
+export const getPVCMetric = (pvc: K8sResourceKind, metric: string): number => {
```

Comment:
I know that this is based on existing patterns in this file, but IMO, we should probably just use the redux `useSelector` hook directly where we want to access this value. The rest of these helpers were probably written before we really started using hooks.

Reply:
As mentioned above mixing class components with hooks is not possible.